modules/javafx.controls/src/main/java/javafx/scene/control/skin/TableHeaderRow.java

```diff
@@ -475,7 +464,26 @@ protected NestedTableColumnHeader createRootHeader() {
         return new NestedTableColumnHeader(null);
     }
 
-
+    /**
+     * Shows a menu containing all leaf columns as items.
+     * An item can be selected/deselected to make the corresponding column visible/invisible.
+     *
+     * @implNote This method can be overridden to create and show a custom menu.
+     * @param mouseEvent the {@link MouseEvent} which was generated when the table menu button was pressed
```

Comment:
There's no need to link to method parameters (`MouseEvent`) since they already linked in the method description.

Reply:
okay. Replaced with `@code`.